reference-architecture/vmware-ansible/roles/nfs-server/files/etc-sysconfig-nfs

```diff
@@ -0,0 +1,25 @@
+# This file is managed by Ansible.
```

Comment:
If a template is used here, then you can use `{{ ansible_managed }}` here instead of hard-coding. The value for `ansible_managed` can be changed using the ansible.cfg file.